webapps/ui/tasklist/grunt/config/watch.js

```diff
@@ -42,7 +42,7 @@ module.exports = function(config, watchConf) {
 
   watchConf.tasklist_dependencies = {
     options: options,
-    files: ['<%= pkg.gruntConfig.enTranslationDir %>/**/*.json'],
```

Comment:
I was wondering why is this only in the tasklist watch script. Do you have an idea?

Reply:
I don't know. My best guess is that the watcher use case is not important for relatively static translation tokens. But it's definitely nice-to-have. I created a follow-up to add it for all other Webapps.

Reply:
https://jira.camunda.com/browse/CAM-14847